app/views/admin/delayed_jobs/index.haml

```diff
@@ -8,31 +8,52 @@
   .card
     %table.table.table-striped.table-responsive.mb-0
       %thead
-        %th ID
         %th Job
         %th Queue
-        %th Run at
-        %th Locked by
-        %th Locked at
-        %th Failed at
+        %th Priority
         %th Last error
         %th
         %th
       %tbody
         - @jobs.each do |job|
           %tr
-            %td= job.id
-            %td= job.name.split(' ').first
-            %td= job.queue
-            %td= job.run_at
-            %td= job.locked_by
-            %td= job.locked_at
-            %td= job.failed_at
-            %td= job.last_error
             %td
+              %strong= job.id
+              %br
+              = job.name.split(' ').first
+              - user_id = job.payload_object.job_data.try(:[], 'arguments').try(:first).try(:[], 'user_id')
+              - arguments = job.payload_object.job_data.try(:[], 'arguments').try(:first)
+              - detail = if arguments.is_a?(Hash) then arguments.try(:[], 'report_class') else arguments end
+              - if user_id
+                %br
+                %strong Run by user id:
+                = user_id
+              - if detail
+                %br
+                = detail
+              %br
+              %strong Run after:
+              = job.run_at
+              - if job.locked_by
+                %br
+                %span.icon-cog.text-success
+                %strong Locked by
+                = job.locked_by
+                at
+                = job.locked_at
               - if job.failed_at
-                = link_to admin_delayed_job_path(job), method: :patch, class: 'btn btn-sm btn-info', data: { confirm: 'Are you sure you want to re-queue this job?'} do
-                  %i.icon-settings_backup_restore
+                %br
+                %span.icon-cancel-circled.text-danger
+                %strong Failed:
+                = job.failed_at
+            %td= job.queue
+            %td= job.priority
+            %td
+              .div{style: 'max-height: 200px; overflow-y: scroll;'}= job.last_error
+            %td
+              = link_to admin_delayed_job_path(job), method: :patch, class: 'btn btn-sm btn-info', data: { confirm: 'Are you sure you want to re-queue this job?'} do
```

Comment:
Yeah, I removed that because there are times when the container OOMs or gets killed and the job isn't running, but isn't in a failed state, but is locked so nothing else can pick it up.  Now, at least in those  situations, we can still re-queue it.